assets/prototype/domain/eventEditor/datetimes/DatesList/filterBar/useDatesListFilterState/index.test.ts

```diff
@@ -88,237 +88,239 @@ const datetimes: Datetime[] = [
+		expect(dates[2].isActive).toBe(false);
+		expect(dates[2].isUpcoming).toBe(true);
 	});
-	const dates = result.current.processedDates;
-
-	expect(dates.length).toBe(3);
-	expect(dates[0].isActive).toBe(true);
-	expect(dates[0].isUpcoming).toBe(true);
-	expect(dates[1].isActive).toBe(true);
-	expect(dates[1].isUpcoming).toBe(false);
-	expect(dates[2].isActive).toBe(false);
-	expect(dates[2].isUpcoming).toBe(true);
-});
 
-test('should update processedDates to reflect changes made by invoking setShowDates with upcomingOnly filter', () => {
-	const { result } = renderHook(() => useDatesListFilterState(datetimes));
+	test('should update processedDates to reflect changes made by invoking setShowDates with upcomingOnly filter', () => {
+		const { result } = renderHook(() => useDatesListFilterState(datetimes));
+
+		act(() => {
+			result.current.setShowDates(ShowDates.upcomingOnly);
+		});
+		const dates = result.current.processedDates;
 
-	act(() => {
-		result.current.setShowDates(ShowDates.upcomingOnly);
+		expect(dates.length).toBe(2);
+		expect(dates[0].isUpcoming).toBe(true);
+		expect(dates[1].isUpcoming).toBe(true);
 	});
-	const dates = result.current.processedDates;
 
-	expect(dates.length).toBe(2);
-	expect(dates[0].isUpcoming).toBe(true);
-	expect(dates[1].isUpcoming).toBe(true);
-});
+	test('should update processedDates to reflect changes made by invoking setShowDates with nextActiveUpcomingOnly filter', () => {
+		const { result } = renderHook(() => useDatesListFilterState(datetimes));
 
-test('should update processedDates to reflect changes made by invoking setShowDates with nextActiveUpcomingOnly filter', () => {
-	const { result } = renderHook(() => useDatesListFilterState(datetimes));
+		act(() => {
+			result.current.setShowDates(ShowDates.nextActiveUpcomingOnly);
+		});
+		const dates = result.current.processedDates;
 
-	act(() => {
-		result.current.setShowDates(ShowDates.nextActiveUpcomingOnly);
+		expect(dates.length).toBe(1);
+		expect(dates[0].isActive).toBe(true);
+		expect(dates[0].isUpcoming).toBe(true);
 	});
-	const dates = result.current.processedDates;
 
-	expect(dates.length).toBe(1);
-	expect(dates[0].isActive).toBe(true);
-	expect(dates[0].isUpcoming).toBe(true);
-});
+	test('should update processedDates to reflect changes made by invoking setShowDates with soldOutOnly filter', () => {
+		const { result } = renderHook(() => useDatesListFilterState(datetimes));
 
-test('should update processedDates to reflect changes made by invoking setShowDates with soldOutOnly filter', () => {
-	const { result } = renderHook(() => useDatesListFilterState(datetimes));
+		act(() => {
+			result.current.setShowDates(ShowDates.soldOutOnly);
+		});
+		const dates = result.current.processedDates;
 
-	act(() => {
-		result.current.setShowDates(ShowDates.soldOutOnly);
+		expect(dates.length).toBe(3);
+		expect(dates[0].isSoldOut).toBe(true);
+		expect(dates[1].capacity).toBe(20);
+		expect(dates[1].sold).toBe(20);
```

Comment:
were these keys supposed to be different? Or should they all be testing the same datetime?
seems unexpected to be testing one property on one datetime and then a couple of different properties on another datetime.
not a big deal and doesn't need to change, just want to ensure that this is as intended

Reply:
Removed that block since it has more atomic approach here: `assets/prototype/domain/shared/predicates/datetimes/filters/soldOutOnly/index.test.ts`
***
https://github.com/eventespresso/event-espresso-core/pull/2174/commits/d93c75fa4c8238c2802775a4125cb2b19fbb10d1